### DISCUSSION TITLE
Premier jet

### DISCUSSION THREAD

**Discussion by giscus** [caezd/lemnos#8] [General]
# Premier jet

wombat profil javascript Description Wombat est une visionneuse de profil qui permet de charger et d'ouvrir le profil des membres directeme

https://lemnos2.forumactif.com/t13-premier-jet#13

**Comment by caezd** [comment]
```diff
public class Hello1
{
   public static void Main()
   {
-      System.Console.WriteLine("Hello, World!");
+      System.Console.WriteLine("Rock all night long!");
   }
}
```

**Reply by caezd** [reply]
test
